cartridges/int_cloudinary/cartridge/js/cloudinaryWidgets.js

```diff
@@ -64,7 +64,7 @@ window.renderCloudinaryVideoPlayer = function () {
 window.makeCloudinaryImagesResponsive = function () {
     var $cldResponsiveImgTags = $('.cld-responsive');
     if ($cldResponsiveImgTags && $cldResponsiveImgTags.length > 0) {
-        window.cldObj = window.cldObj || cloudinary.Cloudinary.new(); // eslint-disable-line no-undef
+        window.cldObj = cloudinary.default.Cloudinary.new(); // eslint-disable-line no-undef
```

Comment:
why are we no longer allowing the previous window.cldObj?

Reply:
after updating the JS version window.cldObj is always undefined

Reply:
if this is never defined, then please use a local variable like with all other widgets

Reply:
the other widget cldObj has the custom object, in this case, we are using an object from the cloudinary side, I can revert back the change if you really think

Reply:
yes, let's revert, so if it will be called twice in the future it will not instantiate additional objects